### PR TITLE
feat(): add rps-avg to reports 

### PR DESCRIPTION
**Pull request by OrFrenkelZooz**

### CODE REVIEW

File: tests/unit-tests/reporter/models/reportsManager-test.js [hunk 1]
```diff
@@ -379,6 +379,45 @@ describe('Reports manager tests', function () {
...
+            const now = new Date();
+            const tenSecBefore = new Date(now).setSeconds(now.getSeconds() - 10);
+            const subscriber = { last_stats: { rps: { total_count: 200 }, codes: { '200': 10 } } };
+            const report = Object.assign({}, REPORT, { last_updated_at: now, start_time: tenSecBefore, subscribers: [subscriber] });
+            databaseGetLastReportsStub.resolves([report]);
+            const reports = await manager.getLastReports();
+            reports.length.should.eql(1);
+            should(reports[0].avg_rps).eql(20);
+        });
+        it('get last report with avg rsp when test finished', async () => {
```
Comment (NivLipetz) [inline tests/unit-tests/reporter/models/reportsManager-test.js:393]: add another unit test case when test run is aborted

File: src/reports/models/statsManager.js [hunk 2]
```diff
@@ -31,6 +32,17 @@ module.exports.postStats = async (report, stats) => {
     return stats;
 };
 
+async function updateSubscriberWithStatsInternal(report, stats) {
+    const parseData = JSON.parse(stats.data);
+    const subscriber = report.subscribers.find(subscriber => subscriber.runner_id === stats.runner_id);
+    const { last_stats } = subscriber;
+    if (last_stats && parseData.rps) {
+        const lastTotalCount = _.get(last_stats, 'rps.total_count') || 0;
```
Comment (NivLipetz) [inline src/reports/models/statsManager.js:40]:
```suggestion
        const lastTotalCount = _.get(last_stats, 'rps.total_count', 0);
```

see https://lodash.com/docs/4.17.15#get

Comment (OrFrenkelZooz) [reply src/reports/models/statsManager.js:40]: cool

File: tests/unit-tests/reporter/models/reportsManager-test.js (hunk 1 above)
Comment (OrFrenkelZooz) [reply tests/unit-tests/reporter/models/reportsManager-test.js:393]: fixed

**Pull request merged**